biome.json

```diff
@@ -1,7 +1,28 @@
 {
   "$schema": "https://biomejs.dev/schemas/1.4.0/schema.json",
+  "files": {
+    "ignore": [
+      "**/dist",
+      "**/node_modules",
+      "**/fixtures",
```

Comment:
Is there a reason why we ignore the fixtures?

Reply:
@ematipico there was a reason, but I can't remember it.
I'm happy to enable it for fixtures :)